HubFramework.xcodeproj/project.pbxproj

```diff
@@ -1473,6 +2161,121 @@
 			};
 			name = Release;
 		};
+		8AE6C0051DF6E3110063B2B1 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
```

Comment:
All of these should preferably live in the, or another, xcconfig file. Since our spotify_os file is a bit… unwieldy… we can do this change in a subsequent PR. It would basically be nice to split it up. Or use a [better set of configs](https://github.com/jspahrsummers/xcconfigs) for the framework, live and demo targets. I’ll take that task.